CMSgov/managed-care-review
# Hw doc categorize cms

## Summary

- User views documents categorized as rate and/or contract supporting on review & submit page and submission summary page
- User knows when a document is listed under both rate and contract categories  

#### Related issues
https://qmacbis.atlassian.net/browse/OY2-10583
https://qmacbis.atlassian.net/browse/OY2-12894

#### Screenshots
 
![Screen Shot 2022-02-01 at 4 54 28 PM](https://user-images.githubusercontent.com/10750442/152065140-b8d4d946-b3af-40f9-95c0-8a4400b391a0.png)

#### Test cases covered

- Cypress: added check for submission summary page that appropriate tables are displayed
- Jest: 
   - Contract details summary renders supporting docs when they exist
   - Rate details summary renders supporting docs when they exist, 
   - Upload table renders documents that are both contract and rate related (with asterisk when needed)
   - Upload table renders documents that are just contract related
   - Upload table render documents that are just rates related 

## QA guidance
- Go through different variations of document categories, rate exclusive, contract exclusive, rate and contract both, not categories assigned (this is really a use case we only need to develop for because we don't have feature flags )
